packages/waku/src/lib/middleware/types.ts

```diff
@@ -4,10 +4,10 @@ import type { EntriesDev, EntriesPrd } from '../../server.js';
 export type ClonableModuleNode = { url: string; file: string };
 
 export type HandlerReq = {
-  body: ReadableStream | null;
-  url: URL;
-  method: string;
-  headers: Record<string, string>;
+  readonly body: ReadableStream | null;
+  readonly url: URL;
+  readonly method: string;
+  readonly headers: Readonly<Record<string, string>>;
 };
```

Comment:
Why not use the `DeepReadonly` type here?

Reply:
nice call, hm, but this looks fine. just preference?

Reply:
yeah this looks fine to me too, I was just curious.